scripts/convert_music_spectrogram_to_diffusers.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+import argparse
+import os
+
+import jax
+import tensorflow as tf
+import torch
+
+from t5x import checkpoints
+from music_spectrogram_diffusion import inference
+from transformers import T5Config
+
+from diffusers import DDPMScheduler, ContinuousContextTransformer, SpectrogramDiffusionPipeline
+
+MODEL = "base_with_context"
+
+
+def main(args):
+    t5_checkpoint = checkpoints.load_t5x_checkpoint(args.checkpoint_path)
```

Comment:
I guess this checkpoint needs to be converted to PyTorch and loaded into the model somewhere no? :-) Looks very nice already though!